tests/end_to_end_cases/operations_api.rs

```diff
@@ -43,17 +45,20 @@ async fn test_operations() {
     let id = operation.name.parse().expect("not an integer");
 
     // Check we can fetch metadata for Operation
-    let fetched = operations_client
-        .get_operation(id)
-        .await
-        .expect("get operation failed");
-    let meta = get_operation_metadata(fetched.metadata);
+    // Todo: remove
+    // let fetched = operations_client
+    //     .get_operation(id)
+    //     .await
+    //     .expect("get operation failed");
+    // let meta = get_operation_metadata(fetched.metadata);
+
+    let meta = operations_client.operation_metadata(id).await;
```

Comment:
Here is the API i was hoping for:

```suggestion
    let meta = operations_client
        .get_operation(id)
        .operation_metadata(id)
        .await
        .metadata();
```

Reply:
Addressed